NAMESPACE

```diff
@@ -607,6 +610,8 @@ importFrom(bridgesampling,bayes_factor)
 importFrom(bridgesampling,bridge_sampler)
 importFrom(bridgesampling,post_prob)
 importFrom(coda,as.mcmc)
+importFrom(extraDistr,dbbinom)
```

Comment:
Since you seem to access these function via `::` to we need to import them?

Reply:
Having it in Suggests is the most sensible thing to do I think. Perhaps for the functions that require a package unders Suggests could be starting with a `brms:::require_package(...)` call. I will just add it manually to your PR.